examples/using-guidebot-flags.md

```diff
@@ -0,0 +1,106 @@
+# Using Switch Cases with Guidebot
+
+This page is an explanation of how to use switch cases with a feature in Guidebot, `message.flags`.
+
+## An Example
+
+Switch cases are a nice feature that can be used without the `message.flags` feature, and can be used like this. 
+
+```js
+// This will look for the first argument. If your command was 'hi', and you did 'hi send', it would send 'Hi!' to the channel.
+const trigger = args[0];
+
+switch (args[0]) {  
+  case 'send': {
+    // This will send in the channel that the command was run in.
+    message.channel.send('Hi!');
+    break;
+  }
+
+  case 'dm': {
+    // This will DM the message author..
+    message.author.send('Hi!');
+  }
+}
+```
+
+However, you can use the switch case with `message.flags`.
+
+```js
+  // This will look for the first argument beginning with -, the flag. If your command was 'hi', and you did 'hi -send', it would send 'Hi!' to the channel.
+switch (message.flags[0]) {
+  case 'send': {
+    // This will send in the channel that the command was run in.
+    message.channel.send('Hi!');
+    break;
+  }
+
+  case 'dm': {
+    // This will DM the message author.
+    message.author.send('Hi!');
+  }
+}
+```
+
+## Using with Arguments
+
+Let's say you're trying to make a command to add or take a role to/from a user. This can be consolidated into one command using `message.flags`. This command is based off of the [Guidebot](https://github.com/AnIdiotsGuide/guidebot) framework, and is called role.
+
+```js
+exports.run = (client, message, args, level) => { // eslint-disable-line no-unused-vars 
+  switch (message.flags[0]) {
+    // This would be 'role -add'.
+    case 'add': {
+      // Check if the message mentions a user.
+      if (message.mentions.users.size === 0) return message.reply('Please mention a user to give the role to.');
+      const member = message.guild.member(message.mentions.member);
```

Comment:
I said [`MessageMentions#members`](https://discord.js.org/#/docs/main/stable/class/MessageMentions?scrollTo=members), that's typeof `Collection<string, GuildMember>`, pretty much very similar to `MessageMentions#users` which returns `Collection<string, User>`. So you need to do `.first()`, and please, check the size of the members collection.

You also don't need `Guild#member(GuildMember)`. And I'm pretty sure it's deprecated as I cannot use it in latest master.

Reply:
Fixed.